next/components/forms/segments/AccountSections/TaxesFeesSection/TaxFooter.tsx

```diff
@@ -0,0 +1,13 @@
+import parse from 'html-react-parser'
```

Comment:
please don't use this

if you need a (somehow) structured text in translation use markdown

if markdown is not enough for what you need, it's ok, just split the translation part into more segments (you can use sub-objects within the translation parts to group related translations together, check out https://github.com/bratislava/bratislava.sk/pull/620/files#diff-06f870440cf0ccbec161a218c290ba04b013aaffd8d69ac65a51b9184caad4a5R31 ), and keep the html/css/react part in code outside of translations